core/jazz_metrics/index.js

```diff
@@ -337,6 +374,146 @@ function cloudWatchDetails(assetParam) {
   });
 }
 
+/*
+* Prepare to obtain azure metrics definiations
+*/
+function azureMetricDefinitions(config, assetParam) {
```

Comment:
This would be the right time to separate Azure related stuff to its own module as `index.js` is starting to get too big.

Reply:
This file, index.js, isn't too long, comparing other files. I think leaving the major two functions there makes read code easier.